components/form/builder/src/index.js

```diff
@@ -0,0 +1,279 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import Select from './components/select'
+import ButtonGroup from './components/button-group'
+import Input from './components/input'
+import TextArea from './components/text-area'
+import AtomButtom from '@schibstedspain/sui-atom-button'
+
+const INITIAL_LIST_ITEM_STATE = []
+const INITIAL_INPUT_ITEM_STATE = ''
+
+const BASE_CLASS = 'sui-FormBuilder'
+const FORM_WRAP_CLASS = `${BASE_CLASS}-wrap`
+const FORM_ITEM_CLASS = `${BASE_CLASS}-item`
+const FORM_ITEM_DISABLED_CLASS = `is-disabled`
+
+class FormBuilder extends Component {
+  constructor({config, submitText}) {
+    super()
+    this._config = config
+    this._formFields = Object.keys(this._config)
+    this._initialState = this._getDefaultInitialState(this._formFields)
+    this._submitText = submitText
+
+    this.state = {...this._initialState, showErrors: false}
+  }
+
+  /* Requested the list of vehicle brands by default at the start of the PTA */
```

Comment:
These kind of comments are so weirds, I suggest you to use JSDoc format for them, like this styleguide does: https://react-styleguidist.js.org/docs/documenting.html#code-comments-and-proptypes

Reply:
Ok, thx!

Reply:
DONE!